client/lib/plans/constants.js

```diff
@@ -512,7 +512,7 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_UPLOAD_PLUGINS ]: {
 		getSlug: () => FEATURE_UPLOAD_PLUGINS,
-		getTitle: () => i18n.translate( 'Install plugins' ),
+		getTitle: () => i18n.translate( 'Install Plugins' ),
```

Comment:
Warning: 22 translations will be lost with this change.
Alternate string suggestion: 
* `translate( 'Installed Plugins' )`&mdash; translations: **77**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)